server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -129,6 +132,12 @@ export class DeliLambda extends EventEmitter implements IPartitionLambda {
     // when set, messages will be nacked based on the provided info
     private nackMessages: INackMessagesControlMessageContents | undefined;
 
+    // Session level properties
+    private scribeStartSuccess: boolean = false;
```

Comment:
Since deli could restart randomly during a session, it could lose the state of this variable. I believe we should add this to the checkpoint data.
@tanviraumi what do you think?

Reply:
Done. I thought of doing this in the next PR but I guess it makes more sense to do it in this one.